packages/coding-standard/src/Rules/NoProtectedElementInFinalClassRule.php

```diff
@@ -100,11 +86,12 @@ private function isPropertyExistInTraits(Class_ $class, string $propertyName): b
     private function isMethodExistInTraits(Class_ $class, string $methodName): bool
     {
         /** @var Identifier $name */
-        $name = $class->name;
+        $name = $class->namespacedName;
```

Comment:
@samsonasik So this was the responsible line, in case you'd be curious.

- `name` returns only short class name - that's why the reflection bellow didn't work
- `namespacedName` return FQN name

Reply:
thank you :+1: